DEVELOPMENT.md

```diff
@@ -3,6 +3,13 @@
 We use [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) to scaffold the kubernetes controllers.
 The [Kubebuilder Book](https://book.kubebuilder.io/) is a good introduction to the topic and we recommend reading it before proceeding.
 
+Please install kubebuilder `3.15.1` before upgrading or adding new Custom Resources:
```

Comment:
should this not be a part of the asdf .tool-versions? 

also slight side note, I wonder if we could just have a single `make` command to add all the necessary plugins and run the asdf install? It's always been annoying to have to go and search for the asdf plugin repo to install it

Reply:
I originally did, but it caused the PR checks to fail, and as its not strictly needed for anything CI related, thought it should just be part of a local development setup. On that note, I agree that would be good, maybe we should include that as part of https://github.com/sky-uk/kfp-operator/issues/380?

Reply:
updated the ticket to include it